domains/vijay.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Vijay's profile",
+    "repo": "https://github.com/hack-e-d/profile.github.io",
+    "owner": {
+        "username": "hack-e-d",
+        "email": "pvvijayraj1998@gmail.com",
+    },
+    "record": {
+        "CNAME": "hack-e-d.github.io"

```

You can either change the repository name to `hack-e-d.github.io` or change the JSON to:
```suggestion
{
    "description": "Vijay's profile",
    "repo": "https://github.com/hack-e-d/profile.github.io",
    "owner": {
        "username": "hack-e-d",
        "email": "pvvijayraj1998@gmail.com",
    },
    "record": {
        "URL": "https://hack-e-d.github.io/profile"
    }
}
```